src/libraries/System.Private.CoreLib/src/System/Reflection/AssemblyNameFormatter.cs

```diff
@@ -15,31 +15,31 @@ public static string ComputeDisplayName(string name, Version? version, string? c
             const int PUBLIC_KEY_TOKEN_LEN = 8;
             Debug.Assert(name.Length != 0);
 
-            StringBuilder sb = new StringBuilder();
-            sb.AppendQuoted(name);
+            var vsb = new ValueStringBuilder(stackalloc char[256]);
```

Comment:
Will it grow when length exceed 256 characters? e.g. `new AssemblyName { Name = new string ('a', 500) }`

Reply:
Yes